### DISCUSSION TITLE
kOps 1.23.1-version

### DISCUSSION THREAD

**Discussion by argoyle** [eddycharly/terraform-provider-kops#554] [General]
Any 1.23.1-version in the works soon?

**Comment by eddycharly** [comment]
All work is already done, going to release this afternoon :)

**Reply by argoyle** [reply]
Perfect! Thanks!

**Reply by eddycharly** [reply]
It's out https://registry.terraform.io/providers/eddycharly/kops/latest/docs

**Reply by eddycharly** [reply]
Please let me know if you observe any issue.

**Reply by argoyle** [reply]
Provider works fine. Migration of existing AWS LBC ingresses from 1.21.x to 1.23.x not so much but that will be an issue on the kOps repo. 😄 Thanks!

**Reply by eddycharly** [reply]
Thanks for trying it out. What was the issue with aws lbc ?

**Reply by argoyle** [reply]
Seems to be unable to add tags on load balancer, listeners, target groups and security groups for the existing resources with the updated IRSA policy. Adding the required tags manually make the aws-loadbalancer-controller silent again.

**Reply by argoyle** [reply]
New clusters and deploying the same deployment/service/ingress works perfectly so just upgrading existing clusters having problems.
